### PR TITLE
Fix the agenda and add two new speakers

### PR DESCRIPTION
**Pull request by GiorgioNatili**

### CODE REVIEW

File: _data/sessions.yml [hunk 1]
```diff
@@ -383,6 +383,33 @@
...
+  subtype: workshop
+  complexity: "intermediate"
+  speakers: [50]
+-
+  id: 029
+  title: "Training CoreML With Your Mobile Camera During A Conference Talk"
+  description: |
+   This will be a collaborative experience - come help David pick some objects to train a CoreML model on stage! We'll use an iPhone camera to take some photos, then we'll use IBM Watson to train the model. After that, with (literally) one line of code, we'll deploy the model into an iOS framework called Lumina to test the model on stage with all of the different objects!
+  outcome: |
+    - How to create a coreML model with nothing but photos and text labels
```
Comment (joleary1987) [inline _data/sessions.yml]: CoreML should be capitalized

File: _data/sessions.yml [hunk 2]
```diff
@@ -383,6 +383,33 @@
...
+    - How to integrate a coreML model into an iOS app
```
Comment (joleary1987) [inline _data/sessions.yml]: Same as above

**Review by joleary1987 — CHANGES_REQUESTED** [review]

File: _data/sessions.yml (hunk 1 above)
Comment (ZevEisenberg) [reply _data/sessions.yml]: It also needs a space: Core ML

File: _data/sessions.yml (hunk 2 above)
Comment (GiorgioNatili) [reply _data/sessions.yml]: Both fixed

**Review by joleary1987 — APPROVED** [review]

**Pull request merged**